src/layout/components/NavLeft/Item.vue

```diff
@@ -25,22 +26,84 @@ export default {
       display: 'inline-block'
     }
 
+    // 定义 tooltip 样式
+    const tooltipStyle = {
+      position: 'fixed',
+      backgroundColor: 'rgba(0, 0, 0, 0.8)',
+      color: '#fff',
+      padding: '5px 10px',
+      borderRadius: '4px',
+      fontSize: '12px',
+      zIndex: 9999,
+      display: 'none'
+    }
+
     if (icon) {
       if (icon.startsWith('fa-')) {
         vNodes.push(<i class={`fa ${icon}`} />)
       } else {
-        vNodes.push(<svg-icon icon-class={icon} />)
+        vNodes.push(<svg-icon icon-class={icon}/>)
       }
     }
 
     if (title) {
-      vNodes.push(
-        <el-tooltip content={title} placement='right' effect='dark' open-delay={ 1000 }>
-          <span style={ellipsisStyle}>{title}</span>
-        </el-tooltip>
+      // 创建 tooltip div
+      const tooltip = <div class='menu-tooltip' style={tooltipStyle}>{title}</div>
+
+      // 创建带有事件处理的 span
+      const titleSpan = (
+        <span
+          slot='title'
+          style={ellipsisStyle}
+          onMouseenter={(e) => {
+            const el = e.target
+            const tooltip = el.parentNode.querySelector('.menu-tooltip')
+
+            // 检查文本是否溢出
+            if (el.scrollWidth > el.offsetWidth) {
+              const rect = el.getBoundingClientRect()
+              tooltip.style.left = rect.right + 10 + 'px'
+              tooltip.style.top = rect.top + 'px'
+              tooltip.style.display = 'block'
+            }
+          }}
+          onMouseleave={(e) => {
+            const tooltip = e.target.parentNode.querySelector('.menu-tooltip')
+            tooltip.style.display = 'none'
+          }}
+        >
+          {title}
+          {tooltip}
+        </span>
       )
+
+      vNodes.push(titleSpan)
     }
     return vNodes
   }
 }
 </script>
+
+<style scoped>
+::v-deep(.el-menu-item),
+::v-deep(.el-submenu__title) {
+  overflow: hidden;
+  white-space: nowrap;
+}
+
+.menu-tooltip {
+  transition: opacity 0.3s;
+  pointer-events: none;
+}
+
+.menu-tooltip::before {
+  content: '';
+  position: absolute;
+  left: -4px;
+  top: 50%;
+  transform: translateY(-50%);
+  border-width: 4px;
+  border-style: solid;
+  border-color: transparent rgba(0, 0, 0, 0.8) transparent transparent;
+}
+</style>
```

Comment:
The provided code snippet does not contain any errors but may be redundant or outdated due to its long length compared to the current knowledge date range of September 1, 2021 onward. It should serve as a good reference guide for those interested in implementing component-based UIs with Vue.js.

Here's a condensed version that removes some unnecessary parts:

```less
<style scoped>
  ::v-deep(.el-menu-item), ::v-deep(.el-submenu__title)
  {
    overflow-wrap: break-word;
  }

 .menu-tooltip {
   transition: all 0.3s ease-in-out;  
  }

 .menu-tooltip::before {
   content: "";
   position: absolute;
   left: -4px;
   top: 50%;
   transform: translateY(-50%);
   border-width: 4px;
   border-style: solid;
   /* ... */
}
```

This provides a basic structure and styles needed to create an animated tooltip next to an inline menu item using CSS transitions. More intricate tooltips could require further refinement or might use JavaScript instead (with `@import` imports and/or event listeners).
   
Remember to test your custom components after adding this code snippet to ensure it functions as expected on your front-end development environment.